vertical-pod-autoscaler/pkg/utils/metrics/quality/quality.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package quality (aka metrics_quality) - code for VPA quality metrics
+package quality
+
+import (
+	"strconv"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/metrics"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	metricsNamespace = metrics.TopMetricsNamespace + "quality"
+)
+
+var (
+	usageRecommendationRelativeDiff = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: metricsNamespace,
+			Name:      "usage_recommendation_relative_diffs",
```

Comment:
Are you planning to also add absolute diffs (usage - recommendation) and recommendation values in some upcoming PR? It would give more information about the observed diff values.

Reply:
Yes, I have exactly this and it is almost ready, will send today :)